drivers/mssqql.go

```diff
@@ -275,22 +277,58 @@ func (db *MSSQL) GetRecords(database, table, where, sort string, offset, limit i
 		if err := rows.Scan(rowValues...); err != nil {
 			return nil, 0, err
 		}
+		// Get column types to identify UNIQUEIDENTIFIER
+		columnTypes, err := rows.ColumnTypes()
+		if err != nil {
+			return nil, 0, err
+		}
 
+		if len(columnTypes) != len(rowValues) {
+			return nil, 0, errors.New("unexported number of column")
```

Comment:
oups, my bad I introduced a typo


```suggestion
			return nil, 0, errors.New("unexpected number of column")
```